BepInEx.Core/Contract/Attributes.cs

```diff
@@ -21,11 +21,15 @@ public class BepInPlugin : Attribute
     /// <param name="GUID">The unique identifier of the plugin. Should not change between plugin versions.</param>
     /// <param name="Name">The user friendly name of the plugin. Is able to be changed between versions.</param>
     /// <param name="Version">The specific version of the plugin.</param>
-    public BepInPlugin(string GUID, string Name, string Version)
+    /// <param name="Author">The author of the plugin.</param>
+    /// <param name="Link">The link to the plugin's website or repository.</param>
+    public BepInPlugin(string GUID, string Name, string Version, string Author = null, string Link = null)
```

Comment:
breaks binary compatibility, also makes no sense to do this since we already have a lot of mods which use ResonitePlugin from the shim, and you can't just delete the attribute from the shim without breaking all already released mods.